### DISCUSSION TITLE
Vercel alias for deployment not propagating correctly

### DISCUSSION THREAD

**Discussion by abdalem** [vercel/community#4296] [Help]
### Summary

Hi everyone !

We currently testing Vercel for deploying and hosting the front of our customers apps. So basically we are on a hooby plan for now.

Everything worked fine except one thing. I Have added a domain to the projet, and making it an alias each time I deploy but the domain is not serving the right deployment. 

It worked fine the first time, but the next deployments were not propagating to my domain (eg: app.domain.dev), and this url keep serving the first deployment I made even that I changed the alias. On the Vercel dashboard when I click on the preview deployment, I can see that my alias was linked to the right deployment but the url keep serving the first one. Did I miss something ? Maybe unlinking first the domain ? And I don't want to link a branch as we want to control the deployment through terraform and our CI, so I need to do it this way because we want in the future to have app.domain.dev as our staging and ${PR_NUMBER}.domain.dev as our pull request preview url. 

### Steps to Reproduce

Basically:
```bash
vercel deploy
#awaiting for the preview url to show up
vercel alias newly_deployed_preview_url app.domain.dev
```
